src/App.js

```diff
@@ -7,7 +9,7 @@ class App extends React.Component {
   render() {
     return (
       <div>
-        <h2>Welcome to your Todo App!</h2>
+        <TodoList />
```

Comment:
so much clean. But think about it this way, if your App component isn't using state, does it need to be a class or can it just be a function? Think about this while you're coding and I highly recommend you get into the mode of refactoring things like this as it will help optimize your code and make it faster and easier to read.